climada/engine/impact.py

```diff
@@ -19,7 +19,7 @@
 Define Impact and ImpactFreqCurve classes.
 """
 
-__all__ = ['ImpactFreqCurve', 'Impact']
+__all__ = ['ImpactFreqCurve', 'Impact', 'ImpactCalc']
```

Comment:
I think this file is too large. I suggest splitting it into two files `impact.py` and `impact_calc.py` inside a submodule `engince/impact/`. The respective `__init__.py` file could already load the resepctive classes, so that this change is fully backwards compatible:
```python
# climada/enginve/impact/__init__.py

from .impact import Impact, ImpactFreqCurve
from .impact_calc import ImpactCalc
```
Usage:
```python
# my_script.py

from climada.engine.impact import Impact, ImpactCalc
```

Reply:
Agreed, that is planed.

Reply:
And already done, as far as I see. Thanks! 😊